packages/api-docs-builder/ProjectSettings.ts

```diff
@@ -1,8 +1,23 @@
+import { ComponentClassDefinition } from '@mui-internal/docs-utilities/ComponentClassDefinition';
 import { ComponentInfo, HookInfo } from './buildApiUtils';
 import { CreateTypeScriptProjectOptions } from './utils/createTypeScriptProject';
 import { CreateDescribeablePropSettings } from './utils/createDescribeableProp';
 import { ReactApi as ComponentReactApi } from './ApiBuilders/ComponentApiBuilder';
 import { ReactApi as HookReactApi } from './ApiBuilders/HookApiBuilder';
+import { Slot } from './utils/parseSlotsAndClasses';
+
+export type SortingStrategiesType = {
+  /**
+   * Sort slots items. Setting null result in no sorting (respect the order provided by TS).
```

Comment:
Rephrasing it to be clearer. Same below.

```suggestion
   * Sort slots items. Setting it to `null` keeps the order defined in the source code.
```